modules/oe_content_publication/config/post_updates/00003_create_fields/field.field.node.oe_publication.body.yml

```diff
@@ -0,0 +1,24 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.body
+    - node.type.oe_publication
+  module:
+    - text
+_core:
+  default_config_hash: epQrM0F0rqe-Y_2pNLD5jzqSV_ZAWtwN2rfls_xQIno
```

Comment:
This should be removed.

Reply:
@22Alexandra this is added as requested by @ademarco

Reply:
Alright then.